### PR TITLE
Client errors were not going to STDERR

### LINKED ISSUE #1959: Make `kontena vault read` scripting friendly
In some cases it would be helpful to be able to script reading values from the vault but currently `kontena vault read KEY` outputs multiple lines and in a format that is not easy to parse for scripting. So, two suggested improvements:

1. Add a '--raw' option so that only the value is output to stdout
2. Any errors such as "Not Found" should go to stderr instead of stdout (this really goes for all commands I think)

### PR DESCRIPTION
**Pull request by kke**
Fixes #1959

For example 404 errors were printed to STDOUT. This PR changes the output to STDERR.

Before:

```
$ kontena vault read foofoofoo > foo.txt
$ cat foo.txt
[error] Not found
```

After:

```
$ kontena vault read foofoofoo > foo.txt
[error] Not found
$ cat foo.txt
$
```

### CODE REVIEW

File: cli/lib/kontena/command.rb [hunk 1]
```diff
@@ -69,6 +69,7 @@ def self.load_subcommand(path)
 
   def self.inherited(where)
     where.extend Finalizer
+    super
```
Comment (SpComb) [inline cli/lib/kontena/command.rb]: What happens here?

**Comment by SpComb** [comment]
Checked that the e2e tests still pass :+1:

File: cli/lib/kontena/command.rb (hunk 1 above)
Comment (kke) [inline cli/lib/kontena/command.rb]: Some unrelated local modifications to the file, removed.

Comment (kke) [inline cli/lib/kontena/command.rb]: I was experimenting with something like:
```
class ListCommand < StackCommand
end
```
to replace
```
class Kontena::Cli::Stacks::ListCommand < Kontena::Command
end
```
